lib/config/presets/internal/workarounds.ts

```diff
@@ -118,4 +119,21 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  javaLTSImageVersion: {
+    description: 'Limit Java runtime versions to LTS releases',
```

Comment:
> I have questions on IF/WHEN this becomes a workaround, how would someone override/exclude it Incase they wanted each major release (6-monthly)?
>
> > They would need to add this workaround to `ignoredPresets`

It sounds like this will be a common question once this feature lands in production. Do we want to change our `description` field to cover this question? Like this:

```suggestion
    description: 'Limit Java runtime versions to LTS releases. To get major releases add `javaLTSImageVersion` to the `ignorePresets` array.',
```

We have docs for the `ignorePresets` [^ignorePresets], but it does not mention specific workarounds users may want to ignore.

[^ignorePresets]: https://docs.renovatebot.com/configuration-options/#ignorepresets

Reply:
The problem is... this comment applies to ANY preset within `config:base`. I think maybe it could appear on a Java page instead if we still have one

Reply:
> The problem is... this comment applies to ANY preset within `config:base`. I think maybe it could appear on a Java page instead if we still have one

You're right! Luckily we have a Java page! [^java]

The explanation of the workaround and how to disable it to get major Java updates again can probably live there. 😉 

[^java]: https://docs.renovatebot.com/java/

Reply:
Thanks @HonkingGoose @rarkins - I've incorporated the suggestion into the packageRule description, as well as drafted an update to the `java.md` docs page